Doc/Makefile

```diff
@@ -5,6 +5,7 @@
 
 # You can set these variables from the command line.
 PYTHON       = python3
+VENVDIR      = env
```

Comment:
What's your thinking about changing the name of the venv directory from venv to env?  Making the change would introduce a compatibility issue; even though "make venv" wasn't documented, it's likely being used and, besides leaving a stray "venv" directory around, it would also require current doc builders to change their scripts.